src/commons/locations.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Seattle": "Seattle",
+    "San_Francisco": "San Francisco / Bay",
+    "Los_angeles": "Los Angeles",
+    "Silicon_Valley": "Silicon Valley",
+    "Chicago": "Chicago",
+    "Boston": "Boston",
+    "New_York": "New York",
+    "Austin": "Austin",
+    "Vancouver": "Vancouver",
+    "Kitchener_Waterloo": "Kitchener-Waterloo",
+    "Toronto": "Toronto",
+    "Peel Region": "Peel_Region",
+    "York Region": "York_Region",
+    "Ottawa": "Ottawa",
+    "Montreal": "Montreal",
+    "Quebec_city": "Quebec City"
```

Comment:
Capital C. Also I think the snake case is supposed to be for both sides? Since we want the LHS to be easy to read for devs and the RHS to be the actual text we pass around.